File: helloworld_go/hello-world.go
package main

import (
	"fmt"
	"time"
	"math/rand"
)


func bubble_sort(inputlist [] int){

	swapped := true
	
	for swapped{
		swapped = false

		for i:=1; i<1000; i++{
			if inputlist[i-1]> inputlist[i]{
				inputlist[i-1], inputlist[i] = inputlist[i], inputlist[i-1]
				swapped = true
			}
		}
	}
	fmt.Println(inputlist)

}


func main(){
	var inputlist []int

	for{
		rand.Seed(time.Now().UnixNano())
		for i:=0; i < 1000; i++{
			inputlist = append(inputlist, rand.Intn(1000))
		}

		start:= time.Now()
		bubble_sort(inputlist)
		duration := time.Since(start)
	
		fmt.Println(duration)
	}
}

File: helloworld_python/helloworld.py
import time, random

def bub_sort(inputlist):
    
    swapped = True
    while swapped:
        swapped = False
        for i in range(1,len(inputlist)):
            if inputlist[i-1] > inputlist[i]:
                inputlist[i], inputlist[i-1] = inputlist[i-1], inputlist[i]
                swapped=True
    print(inputlist)

n = 1000

while n > 0:
    inputlist = []
    sortedlist = []

    for _ in range(n):
        inputlist.append(random.randint(0,n))

    start = time.time()
    sortedlist = bub_sort(inputlist)
    end = time.time()

    print(f"{(end-start)*1000}ms")